.gitattributes

```diff
@@ -1,3 +1,5 @@
+* text=auto eol=lf
+
 # Ignore the auto-generated Wasm spectest files from language statistics of wazero repo.
 # https://stackoverflow.com/questions/19052834/is-it-possible-to-exclude-files-from-git-language-statistics
 wasm/internal/spectests/cases/* linguist-vendored
```

Comment:
unrelated: I just realized that this drifted but somehow it's still working 😄

Reply:
cute.. I guess we should delete it. stats stuff are opaque anyway